RELEASE-NOTES.md

```diff
@@ -39,6 +39,7 @@ The Axom project release numbers follow [Semantic Versioning](http://semver.org/
 - Inlet: Added support for deeply nested containers of structs
 - Inlet: Added support for `void` and strings in Lua-defined functions
 - Inlet: Added `get<std::vector<T>>` for retrieving arrays without index information
+- Inlet: SphinxWriter will now document the signature of function callbacks added to a schema
```

Comment:
@joshessman-llnl -- could you please add a docs branch on RTD to better see the proposed changes?